docs/release-notes/7.17.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[release-notes-header-7.17.0]]
+== 7.17
+
+[discrete]
+[[release-notes-7.17.0]]
+=== 7.17.0
+
+[discrete]
+[[bug-fixes-7.17.0]]
+==== Bug fixes and enhancements
+* Adds detailed telemetry statistics for legacy and regular notifications ({pull}123332[#123332], {pull}122472[#122472]).
+* Fixes a bug that changes the message in the *Activity Log* tab when users re-fetch log data for a date range without data ({pull}123039[#123039]).
+* Updates privilege checks on the *Exceptions* tab ({pull}122902[#122902]).
+* Removes leftover alert notifications after a rule is deleted ({pull}122610[#122610]).
+* Enables cross-space telemetry for cases ({pull}122477[#122477]).
+* Updates the *Reporter* column in the Cases table to use usernames instead of full names ({pull}121820[#121820]).
+* Improves endpoint performance and warns users that a trusted application with a wildcard path might experience performance impacts ({pull}120349[#120349]).
+* Pre-configured connectors cannot be used with cases ({pull}120686[#120686]).
```

Comment:
```suggestion
* Removes cases support for pre-configured connectors ({pull}120686[#120686]).
```
Just trying to match the other bullets that start with verbs.

Reply:
In this case, I don't think support ever existed. So the issue description is more of a statement than a bug fix. @jonathan-buttner feel free to correct me if I'm wrong.

Reply:
You're correct @nastasha-solomon, I think a statement here makes sense.